fleet/backend/pktvisor/endpoint.go

```diff
@@ -0,0 +1,216 @@
+package pktvisor
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-kit/kit/endpoint"
+	"github.com/mainflux/mainflux"
+	"github.com/ns1labs/orb/fleet/backend"
+	"github.com/ns1labs/orb/pkg/errors"
+	"github.com/ns1labs/orb/pkg/types"
+)
+
+func listAgentBackendsEndpoint(pkt pktvisorBackend) endpoint.Endpoint {
```

Comment:
As mentioned, this one should be moved back, it is generic and not pktvisor specific

Reply:
Done